File: app/src/main/java/it/manzolo/job/service/bluewatcher/utils/Bluelog.java
package it.manzolo.job.service.bluewatcher.utils;

public class Bluelog {
    private String data;
    private String message;
    private String type;

    public Bluelog(String data, String message, String type) {
        this.data = data;
        this.message = message;
        this.type = type;
    }

    public String getData() {
        return data;
    }

    public String getMessage() {
        return message;
    }

    public String getType() {
        return type;
    }

    public interface logEvents {
        String ERROR = "E";
        String INFO = "I";
        String BROADCAST = "BROADCASTMESSAGE";
    }

}

File: app/src/main/java/it/manzolo/bluetoothwatcher/log/Bluelog.kt
package it.manzolo.bluetoothwatcher.log

class Bluelog(val data: String, val message: String, val type: String) {

    interface logEvents {
        companion object {
            const val DEBUG = "D"
            const val ERROR = "E"
            const val WARNING = "W"
            const val INFO = "I"
            const val BROADCAST = "BROADCASTMESSAGE"
        }
    }
}